terraform/README.md

```diff
@@ -0,0 +1,86 @@
+# **Terraform Folder README**
+
+---
+
+## **Overview**
+
+This folder contains the Terraform scripts that define the serverless infrastructure for the Band Camp Tracker project. The scripts provision resources such as an AWS Lambda function to execute the ETL pipeline, an EventBridge rule to schedule pipeline runs, and an Amazon ECR repository to store the Docker image for the Lambda function. IAM roles and policies are configured to ensure secure and appropriate permissions for each resource. The infrastructure is designed to be efficient, scalable, and maintainable, leveraging serverless technologies to minimize operational overhead.
+
```

Comment:
This is great but I think the overview should mention all aspects of the infrastructure - not just the pipeline